04-GitRStudio.Rmd

```diff
@@ -105,7 +121,7 @@ Now that you have `staged` your changes, we are going to `commit` to the local r
 knitr::include_graphics("images/Git18.PNG")
 ```
 
-Every `commit` must be accompanied by a summary. It can be anything, but it's helpful to make it short, clear, and descriptive so you know what changes are represented in that commit. Regardless of method used to interact with git --- GitHub Desktop, RStudio, command line --- you must always provide a summary when committing.
```

Comment:
Totally should have this comic here: https://xkcd.com/1296/ :grin:

Reply:
Haha. Absolutely. My commit messages suck. I will include this image.